### ISSUE TITLE
Warnings on startup

### ISSUE THREAD

**Issue by scop** [davesteele/gnome-gmail#11]
With current git, Fedora 23:

```shell
$ ./gnomegmail.py 
./gnomegmail.py:35: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk
./gnomegmail.py:37: PyGIWarning: Secret was imported without specifying a version first. Use gi.require_version('Secret', '1') before import to ensure that the right version gets loaded.
  from gi.repository import Secret
./gnomegmail.py:38: PyGIWarning: Notify was imported without specifying a version first. Use gi.require_version('Notify', '0.7') before import to ensure that the right version gets loaded.
  from gi.repository import Notify
./gnomegmail.py:39: PyGIWarning: Wnck was imported without specifying a version first. Use gi.require_version('Wnck', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Wnck
```

**Comment by scop** [comment]
I suppose adding these version checks might help with issues like #8. Specifically, I can reproduce #8 if I have the libwnck, not libwnck3 package installed.

**Comment by llaumgui** [comment]
I fix it on RPM ASAP. Thx.

**Comment by davesteele** [comment]
In master with 7c4f753bc59bfb3b5ac11506e03608a919922ba7

**Comment by davesteele** [comment]
Fixed in 1.9.5.
